llvm/lib/TableGen/TGLexer.cpp

```diff
@@ -156,7 +156,7 @@ int TGLexer::getNextChar() {
       return EOF;
     }
     PrintError(getLoc(),
-               "NUL character is invalid in source; treated as space");
+               "nul character is invalid in source; treated as space");
```

Comment:
Seems like a special case to leave as is

Reply:
Done, and also changes a similar error in Asm lexer to be consistent in wording with this one (null bytes -> NUL character)